### PR TITLE
Correct minor typos

### PR DESCRIPTION
**Pull request by utilForever**
This revision includes:
- Correct minor typos in docs and source codes

## Checklist

- [x] An issue to be fixed by this PR is listed above.
- [x] New tests are added to ensure new features are working. End-to-end tests are usually in the `./frb_example/pure_dart` example, more specifically, `rust/src/api.rs` and `dart/lib/main.dart`.
- [x] The code generator is run and the code is formatted (e.g. via `just refresh_all`).
- [x] If this PR adds/changes features, documentations (in the `./book` folder) are updated.
- [x] CI is passing.

### CODE REVIEW

File: book/theme/highlight.js [hunk 1]
```diff
@@ -576,7 +576,7 @@ contains:[e.BACKSLASH_ESCAPE,n,a]},{begin:'"',end:'"',illegal:"\\n",
...
 ;return{name:"Dart",keywords:{
-keyword:["abstract","as","assert","async","await","break","case","catch","class","const","continue","covariant","default","deferred","do","dynamic","else","enum","export","extends","extension","external","factory","false","final","finally","for","Function","get","hide","if","implements","import","in","inferface","is","late","library","mixin","new","null","on","operator","part","required","rethrow","return","set","show","static","super","switch","sync","this","throw","true","try","typedef","var","void","while","with","yield"],
+keyword:["abstract","as","assert","async","await","break","case","catch","class","const","continue","covariant","default","deferred","do","dynamic","else","enum","export","extends","extension","external","factory","false","final","finally","for","Function","get","hide","if","implements","import","in","interface","is","late","library","mixin","new","null","on","operator","part","required","rethrow","return","set","show","static","super","switch","sync","this","throw","true","try","typedef","var","void","while","with","yield"],
```
Comment (fzyzcjy) [inline book/theme/highlight.js:579]: hmm this looks like 3rd party js

File: frb_example/pure_dart/dart/lib/bridge_definitions.dart [hunk 2]
```diff
@@ -534,9 +534,9 @@ abstract class FlutterRustBridgeExampleSingleBlockTest {
 
   FlutterRustBridgeTaskConstMeta get kUnwrapRustOpaqueConstMeta;
 
-  Object returnNonDropableDartOpaque({required Object opaque, dynamic hint});
+  Object returnNondroppableDartOpaque({required Object opaque, dynamic hint});
```
Comment (fzyzcjy) [inline frb_example/pure_dart/dart/lib/bridge_definitions.dart:537]: hmm...

**Review by fzyzcjy — COMMENTED** [review]
Thanks for fixing typos but some may not be a typo :)

**Comment by fzyzcjy** [comment]
Btw what tool did you use to find typos? Curious about it

File: book/theme/highlight.js (hunk 1 above)
Comment (utilForever) [reply book/theme/highlight.js]: Oops... I rollbacked it. 😵

**Comment by utilForever** [comment]
> Btw what tool did you use to find typos? Curious about it

I used [typos](https://github.com/crate-ci/typos).

File: frb_rust/src/dart_api_dl/include/dart_api.h [hunk 3]
```diff
@@ -1302,7 +1302,7 @@ DART_EXPORT void Dart_DisableHeapSampling();
  *   Dart_DeleteWeakPersistentHandle once the embedder no longer needs it.
  *
  * - The provided callback must not call into the VM and should do as little
- *   work as possible to avoid performance penalities.
+ *   work as possible to avoid performance penalties.
```
Comment (fzyzcjy) [inline frb_rust/src/dart_api_dl/include/dart_api.h:1305]: dart_api_dl is 3rd party IIRC, so we cannot change it

File: frb_rust/src/dart_api_dl/include/dart_tools_api.h [hunk 4]
```diff
@@ -143,7 +143,7 @@ typedef struct {
  * \return Returns a pointer to a Dart_EmbedderInformation structure.
  * The embedder keeps the ownership of the structure and any field in it.
  * The embedder must ensure that the structure will remain valid until the
- * next invokation of the callback.
+ * next invocation of the callback.
```
Comment (fzyzcjy) [inline frb_rust/src/dart_api_dl/include/dart_tools_api.h:146]: and this file

**Review by fzyzcjy — COMMENTED** [review]
LGTM, only two nits

File: frb_rust/src/dart_api_dl/include/dart_api.h (hunk 3 above)
Comment (utilForever) [reply frb_rust/src/dart_api_dl/include/dart_api.h]: Oh, I rollbacked it. 😵

File: frb_rust/src/dart_api_dl/include/dart_tools_api.h (hunk 4 above)
Comment (utilForever) [reply frb_rust/src/dart_api_dl/include/dart_tools_api.h]: I rollbacked it, too. 🧼

**Comment by fzyzcjy** [comment]
@all-contributors please add @utilForever for doc

**Pull request merged**